arcgis-ios-sdk-samples/Scenes/Add an integrated mesh layer/README.md

```diff
@@ -10,7 +10,7 @@ An integrated mesh is used to generate a mesh over a point dataset. They are mos
 
 ## How to use the sample
 
-After launching the sample, watch the integrated mesh layer load in place of the extruded imagery basemap. Navigate around the scene to visualize the high level of detail on the cliffs and valley floor.
+Run the sample and watch the integrated mesh layer load in place. Navigate around the scene to visualize the high level of detail on the buildings.
```

Comment:
I think the change from "after launching" to "run" is unnecessary and "launch" is the former is the preferred phrase.

Reply:
I copied this change from our design doc to keep consistent with other platforms. 😳

Reply:
It may have been overlooked in the design and it's not a huge deal but I think it's preferred.